privaterelay/views.py

```diff
@@ -40,7 +41,7 @@
 
 def home(request):
     if (request.user and not request.user.is_anonymous):
-        return redirect('/accounts/profile/')
+        return redirect(reverse('fxa_login'))
```

Comment:
I'm confused as to why a logged-in user would be redirected to login?

Reply:
Oh whoops. Good catch. Not sure why I put this in there.

Reply:
Updated.